modules/ROOT/pages/api-led-deploy.adoc

```diff
@@ -4,24 +4,23 @@ include::_attributes.adoc[]
 endif::[]
 
 Now that you've created and tested the Mule app that contains your new API from
-within Studio, you can deploy it to the API to CloudHub. First, you will create an API proxy,
+within Studio, you can deploy the API to CloudHub. First, you will create an API proxy,
```

Comment:
FWIW, I think you can remove the "wills"/future tense and it still works just fine.